Azure/azure-cli
# {ACR} az acr artifact-streaming

**Related command**
az acr artifact-streaming create/update/show
az acr artifact-streaming operation show/cancel

**Description**
New ACR feature: Artifact streaming. Allows users to manage the artifact streaming property in their repo and send streaming artifact creation request for any given image in their ACR

**Testing Guide**
Create a registry and push an image to it.
Enable artifact streaming to the repo and push another image to the registry. It should automatically create a streaming artifact.
Create a referrers streaming artifact under any available image (the first one created)

**History Notes**
[ACR] `az acr artifact-streaming`: Add new group to include a subgroup operation and the commands create/show/update
[ACR] `az acr artifact-streaming operation`: Add new group to help the user manage the artifact streaming creation with the commands cancel/show

---

This checklist is used to make sure that common guidelines for a pull request are followed.
